src/main/java/org/jabref/logic/bibtex/comparator/MetaDataDiff.java

```diff
@@ -32,44 +45,44 @@ public static Optional<MetaDataDiff> compare(MetaData originalMetaData, MetaData
     /**
      * @implNote Should be kept in sync with {@link MetaData#equals(Object)}
      */
-    public List<String> getDifferences(PreferencesService preferences) {
-        List<String> changes = new ArrayList<>();
+    public List<Difference> getDifferences(PreferencesService preferences) {
+        List<Difference> changes = new ArrayList<>();
```

Comment:
I suggest using an EnumSet, can be initialized with noneOf.

Reply:
EnumSet is done, but it does not make sense to me to introduce a map to store the localized strings, as they would have to be initialized. But it is probably very rare that Many metadata diffs happen the same time. So initializing all of the strings would be just a waste of memory and time.